### PR TITLE
Registration fixes (from test session)

### PR DESCRIPTION
**Pull request by djpnewton**
- make registration clearer that it is not a login screen
- allow user to toggle visibility of password fields
- do not center password confirmation input text
- make waiting for email verification text clearer
- fix email/password not saved when error dialog pops up

### CODE REVIEW

**Comment by eoliveros** [comment]
> make registration clearer that it is not a login screen 
not sure where to  see this getting the following when i register and click the confirmation email:
image

so its not getting redirected to the app.bitforge.me site. but when i go back to the other screen where there is waiting for email verification screen, it appears to have log me in.

> allow user to toggle visibility of password fields - done
> do not center password confirmation input text - done
> make waiting for email verification text clearer - done

> fix email/password not saved when error dialog pops up - not sure where to see this

**Comment by eoliveros** [comment]
image

may want eyeball included in this page as well to show what the user is typing when updating password.

**Comment by djpnewton** [comment]
> may want eyeball included in this page as well to show what the user is typing when updating password.

good point

**Comment by djpnewton** [comment]
> > make registration clearer that it is not a login screen
> > not sure where to  see this getting the following when i register and click the confirmation email:

`make registration clearer that it is not a login screen` refers to the *Alloy* registration/login screens *not beryllium*

**Comment by djpnewton** [comment]
updated to have show password button on update password screen

File: lib/login.dart [hunk 1]
```diff
@@ -331,6 +418,61 @@ class BronzeRegisterFormState extends State<BronzeRegisterForm> {
...
+  Future<Acct?> _beRegister(
+      BuildContext context, AccountRegistration reg) async {
+    showAlertDialog(context, 'registering..');
+    var res = await beUserRegister(reg);
+    Navigator.pop(context);
+    var acct = await res.when((content) async {
+      var cancelled = false;
+      Acct? acct;
+      showAlertDialog(context,
+          'we are sending you a confirmation email, please follow the link in the email to confirm...',
```
Comment (4855062503) [inline lib/login.dart:441]: Nice.

**Review by 4855062503 — APPROVED** [review]
LGTM.

**Pull request merged**